### ISSUE TITLE
Website Accessibility-WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (.caps)

### ISSUE THREAD

**Issue by A11Ybot-OP** [sass/sass#3229]
#### Issue
`Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Sass: Sass Basics](https://sass-lang.com/guide)
#### Element path
#example-1-scss > h3 > .caps
#### Snippet
`<span class="caps">SCSS</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.33 (foreground color: #cc6699, background color: #f8f8f8, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 96.0.1054.62

**Comment by A11Ybot-OP** [comment]
There are 10 additional places with `colour contrast` accessibility issues.
